docs/running-on-yarn.md

```diff
@@ -154,6 +154,14 @@ To use a custom metrics.properties for the application master and executors, upd
     To request GPU resources from YARN, use: <code>spark.yarn.am.resource.yarn.io/gpu.amount</code>
   </td>
 </tr>
+<tr>
+  <td><code>spark.yarn.applicationType</code></td>
+  <td><code>SPARK</code></td>
+  <td>
+    Defines more specific application types, e.g. <code>SPARK</code>, <code>SPARK-SQL</code>, <code>SPARK-STREAMING</code>,
+    <code>SPARK-MLLIB</code> and <code>SPARK-GRAPH</code>. Please be careful not to exceed 20 characters.
+  </td>
```

Comment:
add the since version to the config - 3.1.0 - see the latest code https://raw.githubusercontent.com/apache/spark/master/docs/running-on-yarn.md